src/vis/sidebar/BrushOptionButtons.tsx

```diff
@@ -8,52 +11,15 @@ interface BrushOptionProps {
 
 export function BrushOptionButtons({ callback, dragMode }: BrushOptionProps) {
   return (
-    <div className="btn-group" role="group">
-      <input
-        checked={dragMode === EScatterSelectSettings.RECTANGLE}
-        onChange={() => callback(EScatterSelectSettings.RECTANGLE)}
-        type="checkbox"
-        className="btn-check"
-        id="rectBrushSelection"
-        autoComplete="off"
-      />
-      <label className="btn btn-outline-primary" htmlFor="rectBrushSelection" title="Rectangular Brush">
-        <i className="far fa-square" />
-      </label>
-
-      <input
-        checked={dragMode === EScatterSelectSettings.LASSO}
-        onChange={() => callback(EScatterSelectSettings.LASSO)}
-        type="checkbox"
-        className="btn-check"
-        id="lassoBrushSelection"
-        autoComplete="off"
-      />
-      <label className="btn btn-outline-primary" htmlFor="lassoBrushSelection" title="Lasso Brush">
-        <i className="fas fa-paint-brush" />
-      </label>
-      <input
-        checked={dragMode === EScatterSelectSettings.ZOOM}
-        onChange={() => callback(EScatterSelectSettings.ZOOM)}
-        type="checkbox"
-        className="btn-check"
-        id="zoomBrushSelection"
-        autoComplete="off"
-      />
-      <label className="btn btn-outline-primary" htmlFor="zoomBrushSelection" title="Zoom">
-        <i className="fas fa-search-plus" />
-      </label>
-      <input
-        checked={dragMode === EScatterSelectSettings.PAN}
-        onChange={() => callback(EScatterSelectSettings.PAN)}
-        type="checkbox"
-        className="btn-check"
-        id="panSelection"
-        autoComplete="off"
-      />
-      <label className="btn btn-outline-primary" htmlFor="panSelection" title="Pan">
-        <i className="fas fa-arrows-alt" />
-      </label>
-    </div>
+    <SegmentedControl
+      value={dragMode}
+      onChange={callback}
+      data={[
+        { label: <FontAwesomeIcon icon={faSquare} />, value: EScatterSelectSettings.RECTANGLE },
+        { label: <FontAwesomeIcon icon={faPaintBrush} />, value: EScatterSelectSettings.LASSO },
+        { label: <FontAwesomeIcon icon={faArrowsAlt} />, value: EScatterSelectSettings.PAN },
+        { label: <FontAwesomeIcon icon={faSearchPlus} />, value: EScatterSelectSettings.ZOOM },
```

Comment:
Please add some Mantine tooltips to the icons. It's currently unclear to the user what they mean.

![grafik](https://user-images.githubusercontent.com/5851088/195288970-a83e1fdb-e46f-4474-b642-35634aa66108.png)

Reply:
done